function/Übungenzufuntionen3.js

```diff
@@ -0,0 +1,20 @@
+const readline = require('readline');
+const rl = readline.createInterface({input: process.stdin, output: process.stdout});
+const prompt = (query) => new Promise((resolve) => rl.question(query, resolve));
+rl.on('close', () => process.exit(0));
+
+async function execute() { // put all your codes here for user inputs
+    function FindMax(num1,num2,num3,num4){
+        return Math.max(num1,num2,num3,num4);
+    }
+
+    console.log(FindMax(7,2,3,4));
```

Comment:
Wenn ich das z.B. mit `console.log(FindMax(7,2));` aufrufe funktioniert es nicht. In der Angabe steht, man soll die Funktion mit *bis zu* 4 Zahlen aufrufen können.

Reply:
* The function is revised to have default values and error is fixed.